plugin/trino-faker/src/main/java/io/trino/plugin/faker/FakerConfig.java

```diff
@@ -45,7 +45,7 @@ public long getDefaultLimit()
         return defaultLimit;
     }
 
-    @Config("faker.default-limit")
+    @Config("faker.default-rowsCount")
```

Comment:
I don't think we should be using camel case in config property names

Reply:
This change also requires updating the docs

Reply:
this one is a mistake - will revert